### DISCUSSION TITLE
Design Tokens

### DISCUSSION THREAD

**Discussion by andy-polhill** [Legal-and-General/canopy#217] [Ideas]
As Canopy increases in adoption we are going to butt up against more and more teams that don't currently use Angular. We fully expect this to be the case and if we continue to scale we may end up supporting multiple frameworks and even operating systems (e.g. iOS, Android).

I believe a good first stage to offer more support might be to look at creating [design tokens](https://css-tricks.com/what-are-design-tokens/). Starting this discussion to see what others think and if anyone has capacity to look into it.

https://amzn.github.io/style-dictionary/#/

**Comment by andy-polhill** [comment]
@owensgit has made a bit of progress in this, use him as a point of contact if you are interested in it.

**Comment by owensgit** [comment]
Created a experimental repo for design tokens using [Style Dictionary](https://github.com/Legal-and-General/canopy/discussions/217#discussioncomment-503259):

https://github.com/Legal-and-General/canopy-design-tokens 

Will open discussions in there regarding that particular library/solution.
